Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -927,6 +936,19 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
             "serif",
             readWriteConstructParamFlags);
 
+    /**
+     * WebKitSettings:math-font-family:
+     *
+     * The font family used as the default for content using a math font.
+     */
+    sObjProperties[PROP_MATH_FONT_FAMILY] =
+        g_param_spec_string(
+            "math-font-family",
+            _("Math font family"),
+            _("The font family used as the default for content using math font."),
+            "",
```

Comment:
Can we make the default nullptr rather than an empty string? Empty strings in public APIs can be pretty confusing.

```suggestion
            nullptr,
```

Of course, make sure the lower layers can actually handle this. If not, you can expose nullptr to the public APIs and convert to empty string internally.